lib/caesar.rb

```diff
@@ -1,6 +1,28 @@
 class Caesar
-  attr_reader :key
-
+  attr_reader :alphabet, :cipher
+  attr_accessor :shift
```

Comment:
This is bad encapsulation. Other classes shouldn't need to read the alphabet or cipher or be able to change the shift. Other classes only interact with this class through to_cipher, from_file, and to_english. How do you know that? From the spec.